substrate/frame/staking/src/asset.rs

```diff
@@ -41,42 +50,78 @@ pub fn total_balance<T: Config>(who: &T::AccountId) -> BalanceOf<T> {
 ///
 /// This includes balance free to stake along with any balance that is already staked.
 pub fn stakeable_balance<T: Config>(who: &T::AccountId) -> BalanceOf<T> {
```

Comment:
Given the fact that `steakable_balance` naming does not map directly to free + staked, I would probably remove this method and replace it with `free_to_stake::<T>(who).saturating_add(staked::<T>(who))` where it's needed.

Reply:
Yeah, it is a legacy name that is frankly confusing. The new ones make a lot more sense.